src/components/textbox/textbox.component.js

```diff
@@ -301,12 +313,26 @@ Textbox.propTypes = {
   characterLimit: PropTypes.string,
   /** Whether to display the character count message in red */
   warnOverLimit: PropTypes.bool,
+  /** Text applied as a hint */
+  hintText: PropTypes.string,
+  /** Specifies the width of the component */
+  width: PropTypes.oneOf([
```

Comment:
We already have a mechanism for width, the `labelWidth` and `inputWidth` props, so I don't think we need this

Reply:
Agree with Sam on this, I also couldn't seem to get it to work the width is always `144px` or `medium` and the prop is being sent down to the DOM (input) element here 

![image](https://user-images.githubusercontent.com/44157880/135256756-110a97c6-b58d-4f6b-b74e-7e1df691b399.png)

Reply:
About the `labelWidth `and `inputWidth`, it is different mechanism, inputWidth seems to only have impact when labelWidth is also passed. `width` property changes (or will change as it does not work on that commit) the width of the input (see image):
![image](https://user-images.githubusercontent.com/87973304/135437131-a5908e21-aeaa-4270-bca0-0c0db7388aae.png)